### DISCUSSION TITLE
My PRO badge has disappeared

### DISCUSSION THREAD

**Discussion by mathislajs** [community/community#43909] [Other Feature Feedback, Questions, & Ideas] [answered]
A few months ago, my PRO badge disappeared from my profile for no reason after I sponsored an organization. I've already tried going into my access preferences settings, with no success. I opened a ticket and it's been over a month since GitHub responded to me.

I got the badge as a student, and my student status is still valid on [education.github.com](https://education.github.com/)

**Comment by bzkarim250** [comment, marked as answer]
GitHub Education Pro badges are typically valid for one year from the date of issue. However, the expiration date can vary depending on the specific program or organization that issued the badge. If you have any concerns or questions about the expiration of your Pro badge, you should contact the organization or program that issued the badge for more information.
